package/lib/src/beam_page.dart

```diff
@@ -43,16 +43,17 @@ class BeamPage extends Page {
   const BeamPage({
     LocalKey? key,
     String? name,
+    this.path,
     required this.child,
     this.title,
-    this.onPopPage = pathSegmentPop,
+    this.onPopPage = _onPopPage,
```

Comment:
This is a very big change for existing code. I think this will be a breaking change.